lib/Rx/Observable/BaseObservable.php

```diff
@@ -429,4 +455,169 @@ function() use (&$map, $observer) {
             return $refCountDisposable;
         });
     }
+
+    /**
+     * @param $value
+     * @return \Rx\Observable\AnonymousObservable
+     */
+    public static function just($value)
+    {
+        return static::fromArray([$value]);
+    }
+
+    /**
+     * Lifts a function to the current Observable and returns a new Observable that when subscribed to will pass
+     * the values of the current Observable through the Operator function.
+     *
+     * @param \Rx\Operator\OperatorInterface $operator
+     * @return \Rx\Observable\AnonymousObservable
+     */
+    public function lift(OperatorInterface $operator)
+    {
+        return new AnonymousObservable(function (ObserverInterface $observer, SchedulerInterface $schedule) use ($operator) {
+            return $operator($this, $observer, $schedule);
+        });
+    }
+
+    /**
+     * Applies an accumulator function over an observable sequence, returning the result of the aggregation as a single element in the result sequence. The specified seed value is used as the initial accumulator value.
+     *
+     * @param callable $accumulator - An accumulator function to be invoked on each element.
+     * @param mixed $seed [optional] - The initial accumulator value.
+     * @return \Rx\Observable\AnonymousObservable - An observable sequence containing a single element with the final accumulator value.
+     */
+    public function reduce($accumulator, $seed = null)
+    {
+        return $this->lift(new ReduceOperator($accumulator, $seed));
+    }
+
+    /**
+     * Returns an observable sequence that contains only distinct contiguous elements according to the keySelector and the comparer.
+     *
+     * @param null $keySelector
+     * @param null $comparer
+     * @return \Rx\Observable\AnonymousObservable
+     */
+    public function distinctUntilChanged($keySelector = null, $comparer = null)
+    {
+        return $this->lift(new DistinctUntilChangedOperator($keySelector, $comparer));
+    }
+
+    /**
+     * @return \Rx\Observable\AnonymousObservable
+     */
+    public static function never()
+    {
+        return (new EmptyObservable())->lift(new NeverOperator());
```

Comment:
I think this is better implemented as a `NeverObservable` instead of an operator.